models/google_play__country_report.sql

```diff
@@ -0,0 +1,150 @@
+with installs as (
+
+    select *
+    from {{ var('stats_installs_country') }}
+
+), ratings as (
+
+    select *
+    from {{ var('stats_ratings_country') }}
+
+
+), store_performance as (
+
+    select *
+    from {{ var('stats_store_performance_country') }}
+
+), install_metrics as (
+
+    select
+        *,
+        -- take the difference later or nah?
+        sum(daily_device_installs) over (partition by country, package_name rows between unbounded preceding and current row) as total_device_installs,
+        sum(daily_device_uninstalls) over (partition by country, package_name rows between unbounded preceding and current row) as total_device_uninstalls
+
+    from installs 
+
+), store_performance_metrics as (
+
+    select
+        *,
+        sum(store_listing_acquisitions) over (partition by country_region, package_name rows between unbounded preceding and current row) as total_store_acquisitions,
+        sum(store_listing_visitors) over (partition by country_region, package_name rows between unbounded preceding and current row) as total_store_visitors
+
+    from store_performance
+
+), country_join as (
+
+    select 
+        coalesce(install_metrics.date_day, ratings.date_day, store_performance_metrics.date_day) as date_day,
+        coalesce(install_metrics.country, ratings.country, store_performance_metrics.country_region) as country,
+        coalesce(install_metrics.package_name, ratings.package_name, store_performance_metrics.package_name) as package_name,
+
+        coalesce(install_metrics.active_device_installs, 0) as active_device_installs,
+        coalesce(install_metrics.daily_device_installs, 0) as daily_device_installs,
+        coalesce(install_metrics.daily_device_uninstalls, 0) as daily_device_uninstalls,
+        coalesce(install_metrics.daily_device_upgrades, 0) as daily_device_upgrades,
+        coalesce(install_metrics.daily_user_installs, 0) as daily_user_installs,
+        coalesce(install_metrics.daily_user_uninstalls, 0) as daily_user_uninstalls,
+        
+        coalesce(install_metrics.install_events, 0) as install_events,
+        coalesce(install_metrics.uninstall_events, 0) as uninstall_events,
+        coalesce(install_metrics.update_events, 0) as update_events,    
+
+        install_metrics.total_unique_user_installs,
+        install_metrics.total_device_installs,
+        install_metrics.total_device_uninstalls,
+
+        ratings.daily_average_rating,
+        ratings.rolling_total_average_rating,
+
+        store_performance_metrics.total_store_acquisitions,
+        store_performance_metrics.total_store_visitors,
+        store_performance_metrics.store_listing_conversion_rate
```

Comment:
I like the idea of breaking the query out into logical sections. Would you be able to separate the sections with a quick comment explaining the section vs just an empty space.

Reply:
Can you apply this same comment to the other models where you have breaks as well.